### PR TITLE
Create shell for kiritan

### PR DESCRIPTION
**Pull request by PeterGuoRuc**

### CODE REVIEW

File: egs/public_dataset/hts/train.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# Copyright 2020 RUC (author: Shuai Guo)
+
+# . ./path.sh
+
+set -e
+
+stage=3
+ 
+
+if [ $stage -le 1 ]; then
```
Comment (ftshijt) [inline egs/public_dataset/hts/train.sh]: do not change what we have for now

File: egs/public_dataset/kiritan/infer.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+# Copyright 2020 RUC (author: Shuai Guo)
+
+# . ./path.sh
+
+set -e
+
+# infer
+echo ============================================================================
+echo "                                infer                                     "
+echo ============================================================================
```
Comment (ftshijt) [inline egs/public_dataset/kiritan/infer.sh]: we do not need this anymore.

File: egs/public_dataset/kiritan/local/train_dev_test_split.sh [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+mv ${data}/alignment/0031* ${data}/alignment/${test}
+mv ${data}/alignment/00* ${data}/alignment/${train}
+
+# process pitch-beat split
+mkdir -p ${data}/pitch_beat_extraction/${dev}
+mkdir -p ${data}/pitch_beat_extraction/${test}
+mkdir -p ${data}/pitch_beat_extraction/${train}
+mv ${data}/pitch_beat_extraction/30 ${data}/pitch_beat_extraction/${dev}/
+mv ${data}/pitch_beat_extraction/31 ${data}/pitch_beat_extraction/${test}/
+mv ${data}/pitch_beat_extraction/{1..29} ${data}/pitch_beat_extraction/${train}/
```
Comment (ftshijt) [inline egs/public_dataset/kiritan/local/train_dev_test_split.sh]: I think we have more data for kiritan?

File: egs/public_dataset/kiritan/run.sh [hunk 4]
```diff
@@ -23,8 +25,10 @@ if [ ${stage} -le 0 ] && [ ${stop_stage} -ge 0 ]; then
   echo =======================
   echo " Stage0: download data "
   echo =======================
-
-  echo ""
+  mkdir -p ${raw_data_dir}
+  # download kiritan dataset from https://zunko.jp/kiridev/login.php, requires a Facebook account due to licensing issues
```
Comment (ftshijt) [inline egs/public_dataset/kiritan/run.sh]: this may echo out

File: egs/public_dataset/kiritan/run.sh [hunk 5]
```diff
@@ -43,7 +47,13 @@ if [ ${stage} -le 2 ] && [ ${stop_stage} -ge 2 ]; then
   echo " Stage2: collect_stats "
   echo =======================
 
-  ${train_cmd} train.py --collect_stats=True -c conf/train_rnn.yaml
+  ${cuda_cmd} --gpu ${ngpu} ${expdir}/stats.log \
+  train.py \
+    -c conf/train_rnn.yaml \
```
Comment (ftshijt) [inline egs/public_dataset/kiritan/run.sh]: I think rnn is not working? Why not use normalized one

File: egs/public_dataset/kiritan/train.sh [hunk 6]
```diff
@@ -0,0 +1,84 @@
+# /usr/bin/bash
+
```
Comment (ftshijt) [inline egs/public_dataset/kiritan/train.sh]: we don't need it anymore

**Review by ftshijt — CHANGES_REQUESTED** [review]

File: egs/public_dataset/kiritan/local/train_dev_test_split.sh [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+mkdir -p ${data}/pitch_beat_extraction/${train}
+mv ${data}/pitch_beat_extraction/30 ${data}/pitch_beat_extraction/${dev}/
+mv ${data}/pitch_beat_extraction/31 ${data}/pitch_beat_extraction/${test}/
+mv ${data}/pitch_beat_extraction/{1..50} ${data}/pitch_beat_extraction/${train}/
+
+# process wav split
+mkdir -p ${data}/wav_info/${dev}
+mkdir -p ${data}/wav_info/${test}
+mkdir -p ${data}/wav_info/${train}
+mv ${data}/wav_info/30 ${data}/wav_info/${dev}/
```
Comment (ftshijt) [inline egs/public_dataset/kiritan/local/train_dev_test_split.sh]: this should follow our conventions (did we use 30 for dev?)

File: egs/public_dataset/kiritan/run.sh [hunk 8]
```diff
@@ -6,9 +6,11 @@
 . ./cmd.sh || exit 1;
 
 
-stage=0
-stop_stage=100
+stage=2
```
Comment (ftshijt) [inline egs/public_dataset/kiritan/run.sh]: change it to 0 and 100 as an initial stage setting

**Review by ftshijt — COMMENTED** [review]
you should change the configuration yaml files as well (relative path recommended).

BTW, fix CI bugs

File: egs/public_dataset/kiritan/run.sh (hunk 8 above)
Comment (PeterGuoRuc) [reply egs/public_dataset/kiritan/run.sh]: ok

**Pull request merged**